student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] You are almost there! However, as the previous reviewer said, it would be better to initialize `age`, `name`, and `parent_permission`. Take a look at this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance).

Reply:
Well resolved